app/models/organization.rb

```diff
@@ -128,6 +128,12 @@ def parent?; child_ids.present? end
 
   def child_organizations; Organization.where(id: child_ids) end
 
+  def nearby_organizations
+    return self.class.none unless regional? && search_coordinates_set?
+    self.class.where.not(id: child_ids).near(search_radius).reorder(id: :asc)
+    # nearbys(search_radius).where.not(id: child_ids)
```

Comment:
This makes tests pass but then fails when actually loading the dashboard page. Any suggestions @jmromer ?

Reply:
Can't repro yet — what kind of a failure are you seeing? (both L133 and L134 work for me)

<img width="1391" alt="Screen Shot 2019-11-20 at 2 55 47 PM" src="https://user-images.githubusercontent.com/4433943/69273130-d9cfb900-0ba5-11ea-895c-bd6cf7b90675.png">

Reply:
@sethherr 
The following works for me:
```rb
    nearbys(search_radius).where.not(id: child_ids).reorder(id: :asc)
```

The `.reorder(id: :asc)` is needed because by default geocoder tries to order by `distance`, which isn't present in this case.

I'm also seeing `self.class.where.not(id: child_ids).near(search_radius).reorder(id: :asc)` work too though.

Reply:
Thanks!

Reply:
See comment https://github.com/bikeindex/bike_index/pull/1424#issuecomment-557053391